packages/react/src/components/SuiteHeader/SuiteHeader.jsx

```diff
@@ -29,7 +31,7 @@ const ROUTE_TYPES = {
   SURVEY: 'SURVEY',
 };
 
-export const SuiteHeaderRoutePropTypes = PropTypes.shape({
```

Comment:
How come you are removing the `PropTypes.shape`? Setting it this way will throw an error, will it not? I believe proptypes need to be a function.

Reply:
Never mind I see you are adding the shape below.

Reply:
@davidicus I did this because I saw other examples where the proptype objects are exported as plain objects, and then the component importing it would define its own proptypes as `PropTypes.shape(ImportedObjectFromAnotherComponent)`. [Here](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/packages/react/src/components/Header/Header.jsx#L18), for example. If this is not a good practice, I can revert the change and export the objects as the return value of `PropTypes.shape()`. Besides, I am not sure why this particular proptype is being exported, nobody imports it.

Reply:
Oh, I forgot to answer your question, it doesn't throw errors when the main component proptypes object is defined because when I removed the `PropTypes.shape()` from te object being exported, I also added the `PropTypes.shape()` when defining the current component's propTypes object:
It was like this before:
```
  /** URLs for various routes on Header buttons and submenus */
  routes: SuiteHeaderRoutePropTypes.isRequired,
```
And now it is defined like this:
```
  /** URLs for various routes on Header buttons and submenus */
  routes: PropTypes.shape(SuiteHeaderRoutePropTypes),
```

Reply:
No, it is fine. I just had missed where you added the `PropTypes.shape` further down the file